src/Tasks/Microsoft.NET.Build.Tasks/PublishAssembliesResolver.cs

```diff
@@ -3,14 +3,17 @@
 
 using System;
 using System.Collections.Generic;
+using System.Collections.Concurrent;
 using System.IO;
 using NuGet.Frameworks;
 using NuGet.ProjectModel;
+using NuGet.Packaging.Core;
 
 namespace Microsoft.NET.Build.Tasks
 {
     public class PublishAssembliesResolver
     {
+        private  HashSet<PackageIdentity> _allResolvedPackages = new HashSet<PackageIdentity>();
```

Comment:
This can be removed now.  It doesn't do anything - the collection is only ever added to, but no one gets the values out.